src/irr/asset/CBufferLoaderBIN.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2009-2012 Gaz Davidson
+// This file is part of the "Irrlicht Engine".
+// For conditions of distribution and use, see copyright notice in irrlicht.h
+
+#pragma once
+
+#include "irr/asset/IAssetLoader.h"
+#include "irr/asset/ICPUMeshBuffer.h"
+
+namespace irr
+{
+	namespace asset
+	{
+		//! Binaryloader capable of loading source code in binary format
+		class CBufferLoaderBIN : public asset::IAssetLoader
```

Comment:
I guess you can write `class final` and remove all the `virtual` keywords from the methods